js/appconfig.ts

```diff
@@ -89,6 +89,7 @@ export class AppConfig {
         useShortMoneyRepresentationForBets: false,
         minConvertibleToSIBetValue: 10000,
         moneySeparator: ",",
+        fractionalDigitsCount: 2,
```

Comment:
By default should be 0 to display all numbers (offline table), for mobile it should be 2

Reply:
0 means dont show any fractional numbers ( 1 253 236 => 1M). For offline table stays useShortMoneyRepresentationForBets as false by default.